src/components/Shifts/AddShiftsForm.tsx

```diff
@@ -31,7 +32,8 @@ export const AddShiftsForm = ({
                                 interval,
                                 shiftForms,
                                 confirmHandler,
-                                isEditingPersistedShift
+                                isEditingPersistedShift,
+                                addSingleShift
```

Comment:
isEditingPersistedShift & addSingleShift are different types of phrase. One is a command and one is a question. We could improve the readability if we changed them to:
- `formMode: 'add' | 'edit'`
- `disableAdd`: boolean

This would make the intent clearer from the caller's perspective